config/graphql/decorators/Teaser.decorator.yaml

```diff
@@ -3,9 +3,9 @@ TeaserDecorator:
   decorator: true
   config:
     fields:
-      url:
-        type: "String!"
-        description: Teaser URL
```

Comment:
I think this will have to stay in until the FH Münster has corrected it. I don't know whether it can be marked as "deprecated".

Reply:
I readded the url field but marked it as deprecated